_pages/API/v1.9/DataService/020-dataservice-releasenotes.md

```diff
@@ -0,0 +1,12 @@
+<h2 id="{{page.sections['dataservice']['secs']['releaseNotes'].anchor}}">{{page.sections['dataservice']['secs']['releaseNotes'].title}}</h2>
+
+<p></p>
+
+{% assign version="1.8.0" %}
```

Comment:
This is the release note section at the top of the page, you would need to increase the version and adapt the feature list below to contain your new parameter.